compass.yml

```diff
@@ -0,0 +1,18 @@
+name: mediator-agent
```

Comment:
What is this file? Do we need it?

Reply:
We don't need this now as we used this for mapping config-as-code for Atlassian